x/nervousresolver/nervousresolver.go

```diff
@@ -63,8 +63,6 @@ type bogonLookup struct {
 // design document on which we're working with Vinicius Fortuna,
 // Jigsaw, aimed at significantly improving OONI measurements quality.
 //
-// See https://docs.google.com/document/d/1jcidvZGxBlucyLivAtvwrCidkIgb3yV6bhQIH_jf21M/edit?ts=5db300e4#
-//
```

Comment:
Rationale: linking to a WIP that we have not made public yet is useless for everyone 🤷